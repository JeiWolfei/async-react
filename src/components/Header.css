.header {
  background-color: cyan;
  font-weight: bold;
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  margin: -8px;
  h1 {
    color: red;
    font-style: italic;
    font-size: 26px;
    margin-top: 5px;
  }
  img {
    width: 300px;
    padding-top: 8px;
    padding-bottom: 0px;
  }
}
