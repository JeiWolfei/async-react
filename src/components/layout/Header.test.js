import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

describe('Header', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <Header title={'<async React app/>'}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
