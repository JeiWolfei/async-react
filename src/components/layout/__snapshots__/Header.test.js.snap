// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches a snapshot 1`] = `
<header
  className="header"
>
  <img
    src="/path/image"
  />
  <h1>
    &lt;async React app/&gt;
  </h1>
</header>
`;
